src/webapp/ldap_data.py

```diff
@@ -91,7 +91,7 @@ def get_email_lookup(yaml_data):
             continue
         for Email in ('PrimaryEmail', 'SecondaryEmail'):
             if Email in ci:
```

Comment:
Don't you also want this too?
```suggestion
            if Email.lower() in ci:
```

Reply:
No - `Email` here loops over the two keys `'PrimaryEmail'` and `'SecondaryEmail'`, so `.lower()` there would look for items under `'primaryemail'` and `'secondaryemail'`, which is not what we want.